### DISCUSSION TITLE
How to reproduce bugs in Codesandbox?

### DISCUSSION THREAD

**Discussion by zvictor** [vercel/next.js#13227] [Help]
I am trying to reproduce a bug in Codesandbox but I came to the conclusion that it must be performing customisations on top of the `next dev` that interfere on the behaviour I want to reproduce.

I noticed  that the exact same code in my machine would show the `Prerendered page` badge, but in Codesandbox it would not. Checking `window.__NEXT_DATA__ ` confirms my suspicions.

### Codesandbox:
![console.log](https://user-images.githubusercontent.com/181076/82664672-7dbc1f00-9c32-11ea-9b3e-8b1879eab5ca.png)

### dev machine:
![console.log](https://user-images.githubusercontent.com/181076/82664776-c5db4180-9c32-11ea-8ce3-42e1c36f81a9.png)

![`Prerendered page` badge](https://user-images.githubusercontent.com/181076/82664290-ae4f8900-9c31-11ea-8f64-c735742643b1.png)
----

Why aren't `autoExport` and `nextExport` present in any of the Codesandbox exports?

**Comment by timneutkens** [comment]
Sounds like it's just an old version of Next.js 🤔

**Reply by zvictor** [reply]
I am using next.js 9.4.2, here is the link: https://codesandbox.io/s/multiple-head-tags-80u9i?file=/pages/index.js
